src/Core/Extensions/EnumExtensions.cs

```diff
@@ -35,4 +35,17 @@ public static class EnumExtensions
 
         return description;
     }
+
+    public static TEnum GetEnumByDescription<TEnum>(this string description) where TEnum : struct, Enum
```

Comment:
Please, allow a nullable description and add an optional argument to ignore case sensitive description.
```csharp
public static TEnum? GetEnumByDescription<TEnum>(this string? description, bool ignoreCase = false)
     where TEnum : struct, Enum
{
    return Enum.GetValues<TEnum>()
                .FirstOrDefault(i => string.Compare(GetDescription(i), description, ignoreCase) == 0);
}
```

Reply:
Sure, I'll make that change in the future commits